### PR TITLE
ref(onboarding): ends multi-select experiment

### PR DESCRIPTION
**Pull request by scefali**
This PR end the multi-select platform experience because it's winning prettily handily. This PR does not remove a lot of the older unused components and we will take care of that in upcoming PRs (right now it would cause a bunch of merge conflicts for other in-flight PRs).

### CODE REVIEW

File: static/app/views/onboarding/onboardingController.tsx [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
-    logExperiment({
-      key: 'TargetedOnboardingMultiSelectExperiment',
-      organization: rest.organization,
-    });
-  }, [rest.organization]);
-  if (rest.organization?.experiments.TargetedOnboardingMultiSelectExperiment) {
-    return <TargetedOnboarding {...rest} />;
-  }
-  return <Onboarding {...rest} />;
+  // TODO: unccoment
```
Comment (Neo-Zhixing) [inline static/app/views/onboarding/onboardingController.tsx:10]: unccoment

**Review by PixelDust22 — APPROVED** [review]

File: static/app/views/onboarding/onboardingController.tsx (hunk 1 above)
Comment (scefali) [reply static/app/views/onboarding/onboardingController.tsx:10]: lol

File: static/app/views/onboarding/onboardingController.tsx [hunk 2]
```diff
@@ -1,24 +1,20 @@
-import {ComponentPropsWithoutRef, useEffect} from 'react';
+import {ComponentPropsWithoutRef} from 'react';
```
Comment (evanpurkhiser) [inline static/app/views/onboarding/onboardingController.tsx:1]: Don't import this, use `React. ComponentPropsWithoutRef`

Comment (scefali) [reply static/app/views/onboarding/onboardingController.tsx:1]: why?

Comment (evanpurkhiser) [reply static/app/views/onboarding/onboardingController.tsx:1]: bc you don't need to import it :)

`React` is in the global type namespace. I would like to add an eslint for this, but haven't looked into it

**Pull request merged**